lib/restforce/config.rb

```diff
@@ -88,7 +88,7 @@ def option(*args)
       end
     end
 
-    option :api_version, default: lambda { ENV['SALESFORCE_API_VERSION'] || '26.0' }
```

Comment:
I'd prefer not to change the default for compatibility reasons, and instead just use a `version_guard` in your methods - see the `#limits` method in `api.rb` for an example.